FEATURE_MATRIX.md

```diff
@@ -13,8 +13,9 @@
 |---|---|---|---|
 | OpenAI                            | ✅ | ✅ | |
 | AzureOpenAI                       | ✅ | ✅ | |
-| Hugging Face                      | ✅ | ❌ | Coming soon to Python - both native and web endpoint support |
-| Custom                            | ✅ | ❌ | Requires the user to define the service schema in their application |
+| Hugging Face Inference API        | 🔄 | ❌ | Coming soon to Python, not all scenarios are covered for .NET |
+| Hugging Face Local                | ❌ | ✅ | |
```

Comment:
Ohhh, not fair. Surely C# can at least get a "N/A" for "Hugging Face Local"

Reply:
We're aiming to effectively support via some combination of TorchSharp, ML.NET, and ONNXRuntime. I think this is fair :)